src/filed.py

```diff
@@ -4,56 +4,54 @@
 
 from os import walk, path, listdir
 
+from predicates.predicate import *
+from actions.action import *
+
 
 class Rule(object):
     src = None
-    dst = None
-    reg = None
+    predicates = None
+    actions = None
 
 
 def parse_config_file(fd):
     """Parses the file descriptor line by line. Skips comments. This also
     validates all the configurations, therefore the return value should be safe
     and usable.
     """
-
     rules = []
 
     for num, line in enumerate(fd):
         line = line.strip()
 
         # Skip comments
-        if line.startswith('#'):
+        if line.startswith('#') or not len(line) > 1:
             continue
 
-        # Format of the line is src:dst:regexpression
+        # See resource/filedrc for example format
         parts = line.split(':')
         if len(parts) != 3:
-            logging.warning("{}:{}: Error reading line".format(filename, num))
+            logging.warning("{}:{}: Error reading line".format(line, num))
```

Comment:
`filename` gives you the config file that is having a problem. The line itself should be part of the error, IMO.